mrp_brewing/models/stock.py

```diff
@@ -37,6 +37,16 @@ def _compute_qty_available(self):
                 qty_available += quant.qty
             lot.qty_available = qty_available
 
+    @api.model
+    def _batch_compute_qty_available(self):

```

Comment:
put the common code in a function that will be called both by _batch_compute_qty_available and _compute_qty_available.

Reply:
Yup, sure thing.